Frameworks/CoreText/CTLine.mm

```diff
@@ -22,25 +22,89 @@
 #import <CoreText/CTTypesetter.h>
 #import <CoreFoundation/CFString.h>
 
-#include <memory>
+#import <memory>
 #import <algorithm>
 #import <numeric>
 #import <cwchar>
 #import <vector>
 
+static inline CFRange __CFRangeIntersectionRange(CFRange a, CFRange b) {
```

Comment:
sadly this guy has to go :(

---
In reply to: [112723252](https://github.com/Microsoft/WinObjC/pull/2586#discussion_r112723252) [](ancestors = 112723252)